### PR TITLE
feat: Project Submissions

### PR DESCRIPTION
**Pull request by jackyzha0**
image

image

# Changelog
* refactor submission to be single page
* revamp judging feedback card to include criteria and to work with new grades schema
* add page to link user to project

### CODE REVIEW

File: src/components/Submission.js [hunk 1]
```diff
@@ -38,48 +59,60 @@ const Column = styled.div`
...
+      {feedback.map((item, i) => {
+        const ordered = {}
+        Object.keys(item)
+          .sort()
+          .forEach(function (key) {
+            ordered[key] = item[key]
+          })
+
+        const grades = Object.entries(ordered).filter(([key, value]) => key !== 'notes')
+        const total = Object.values(grades).reduce((accum, cur) => accum + cur[1], 0)
```
Comment (ianmah) [inline src/components/Submission.js:72]: not a blocker but would prefer this to be done in a container not in the component

File: src/containers/SubmissionLink.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      setMessage(`Syncing Devpost...`)
+
+      // try to find a submission which contains this email
+      const project_ref = await getProjectByEmail(email)
+      if (project_ref === NO_PROJECT) {
+        setMessage(`Not found. Message an organizer in #ask-organizers.`)
+        return
+      }
+
+      const projectId = project_ref.docs[0].id
```
Comment (ianmah) [inline src/containers/SubmissionLink.js:38]:
```suggestion
      const projectRef = await getProjectByEmail(email)
      if (projectRef === NO_PROJECT) {
        setMessage(`Not found. Message an organizer in #ask-organizers.`)
        return
      }

      const projectId = projectRef.docs[0].id
```

**Comment by ianmah** [comment]
@jackyzha0 made some changes feel free to merge if u think they are okkkk

**Review by ianmah — APPROVED** [review]

**Comment by jackyzha0** [comment]
> @jackyzha0 made some changes feel free to merge if u think they are okkkk

looks good, ty!!!

**Pull request merged**